scapy/layers/vrrp.py

```diff
@@ -62,9 +62,16 @@ class VRRPv3(Packet):
         BitField("res", 0, 4),
         BitField("adv", 100, 12),
         XShortField("chksum", None),
-        # FIXME: addrlist should also allow IPv6 addresses :/
-        FieldListField("addrlist", [], IPField("", "0.0.0.0"),
-                       count_from=lambda pkt: pkt.ipcount)]
+        # use addrlist for IPv4, addrlist6 for IPv6
+        ConditionalField(
+            FieldListField("addrlist", [], IPField("", "0.0.0.0"),
+                           count_from=lambda pkt: pkt.ipcount),
+            lambda p: isinstance(p.underlayer, IP)),
+        ConditionalField(
+            FieldListField("addrlist6", [], IP6Field("", "::"),
+                           count_from=lambda pkt: pkt.ipcount),
+            lambda p: isinstance(p.underlayer, IPv6))
```

Comment:
Could you rather use `MultipeTypeField`. This would allow to set a default value in the case that there is no underlayer.

Reply:
@gpotter2 Yes I can. I prepared a patch to do this yesterday in order to fulfill request from @guedou to use the same field name for IPv4 and IPv6 addresses. I am testing now and will update the PR when it is complete.

Reply:
The PR is updated and now uses MultipleTypeField for addrlist.